### ISSUE TITLE
Reshape maps: new improvements

### ISSUE THREAD

**Issue by markotoplak** [Quasars/orange-spectroscopy#576]
Reshape Map is could be improved.

- [ ] Rename the widget. It hints that there already is a map to begin with, but what the widget actually does is to create features with X and Y indices for data given in series. So perhaps **Create Map** would be a better widget. And even if a mapping was there, this widget does not touch it (it does not have an interface for this).
- [ ] Allow column-based ordering (rotations). What the widget does is that it goes through the spectra one by one and assigns indices as it goes through. Because the data can be input in different spectra ordering, the widget needs to allow that the indexing of axes is reversed (done in #573). The widget currently always works row-wise: as the data is read, rows are "filled" first. Add an option for a column-based fill. That option will, effectively, allows setting the output image in all possible rotations and mirroring.
- [ ] After the rename update the documentation.

**Comment by stuart-cls** [comment]
I agree "Create Map" makes more sense. I've also recently needed the ability to set arbitrary x/y values. I've added that to the list above.
